docs/features/firefox-push/development.md

```diff
@@ -0,0 +1,162 @@
+---
+id: development
+title: Development Process
+---
+
+We develop and deploy on a 2 week schedule. A new release is
+tagged and cut a week before deploy, QA files a deployment ticket filed then verifies the release. On occasion a release
+will get a full suite of load-tests as well.
```

Comment:
I'm not sure if this is the right home for this push-server-related documentation; @tublitzed do you have any preferred location where this might be moved to?

Reply:
cc @jrconlin and @pjenvey in case you have thoughts here

Reply:
Yeah, this hits a bit of an ownership issue, which are never great. Technically, Push (like Sync) is "owned" by two teams. The only true, centralized, documentation site mozilla really has is MDN, and I'm not sure that this documentation belongs there. I agree that there should be one source of truth for all mozilla back end services rather than make users hunt around for things. Ideally we should point the various existing docs to that one source of truth. 

That said, I agree with Rachel that this repo is better than any other alternative available, so I'd also like to see the documentation live here. 
![mario](https://user-images.githubusercontent.com/35755/93109758-4d791000-f669-11ea-8765-b5eeda1d501f.png)